src/batou_ext/cron.py

```diff
@@ -128,13 +128,19 @@ class SystemdTimer(batou.component.Component):
 
     """
 
-    _required_params_ = {"command": "/bin/true", "onCalendar": "*/4 * * * * "}
+    _required_params_ = {"command": "/bin/true", "onCalendar": "02:00:00"}
```

Comment:
Why this change?

Reply:
SystemD doesn't use cron syntax. Feel free to use a different example but the old one wouldn't even work.